guppylang/decorator.py

```diff
@@ -291,23 +291,32 @@ def load(self, m: ModuleType | GuppyModule) -> None:
         module = self._modules[caller]
         module.load_all(m)
 
-    def take_module(self, id: ModuleIdentifier | None = None) -> GuppyModule:
```

Comment:
This is a breaking change. I imagine no-one is calling this directly, but we may want to add a `@deprecated` alias `0.10.1` ?